main.less

```diff
@@ -20,7 +20,7 @@
   border-radius: 0.25em;
   display: flex;
   flex-flow: row wrap;
-  padding: 10px 5px;
+  padding: 3.5% 5px;
```

Comment:
This seems problematic, as in, lots of padding on large screens. Why not fixed padding and allow parent layouts that are using this class to deal with padding complexity?